Useful tools & utilities/Figma.md

```diff
@@ -0,0 +1,124 @@
+element. In the Code section, there is a small code snippet which can be used as a guideline for implementing the design in code. On this
+image, Android is selected as the preferred platform in the drop-down menu, however, you can choose iOS or CSS and the code snippets will be
+displayed accordingly. Depending on the selected element, more sections might be present in this Inspect tab, such as sections with
+information about Colors, Borders, Typography, etc. If you hover around the designs and have the button selected, you can see some lines
+which represent its constraints. You can also view the margins, spacing and other dimensions by hovering over particular regions in the
+designs.
+
+### Exports
+
+In the Figma files for the project you'll be working on, there will probably be an Export tab on the right side as well when a certain
+element/component is selected. If you have edit access, you'll have the Export section in the Design tab, however, for developers, it is
+usually enough to have only view access to the Figma files.
+
+The Export tab will look something like this (there are also images from several options provided in this tab which will be discussed
+below):
+
+![Exporting in Figma](/img/figma/figma_exports.png "Exporting in Figma")
+
+In the Export tab, the default export size is 1x and when you click on it, a drop-down list with more options is displayed. You can also
+write the suffix which you would like the exported file to have and choose the image format of the export. Additionally, there is an option
+to set some extra settings if you click on the three dots placed after the image format. If you click on the "+" button, you can export
+multiple files at once (you can set different settings for each of the exported files). The Preview section lets you see how the exported
+file will look like after the export. Here's
+a [link](https://help.figma.com/hc/en-us/articles/360040028114-Guide-to-exports-in-Figma) to a nice guide for exports in
+Figma.
+
+### Comments and discussions
+
+You can enter Comment mode by clicking on the Comment icon in the toolbar. As long as you are in Comment mode, you can add a comment
+anywhere on the canvas by clicking on the spot where you want it to be added.
+
+![Comments in Figma](/img/figma/figma_comment.png "Comments in Figma")
+
+Other team members can join in the discussion by replying to the comment and as it was mentioned before, you can @mention other team members
+in the comments. You can also resolve the comment thread once the discussion is done.
+
+![Replying and resolving comments in Figma](/img/figma/figma_comment_resolve.png "Replying and resolving comments in Figma")
+
+When a comment is resolved, by default, it will no longer appear on the canvas. However, you can customize this and some other settings in
+the options menu shown in the image below:
+
+![Figma comment settings](/img/figma/figma_comment_settings.png "Comment settings in Figma")
+
+
```

Comment:
Do you think it would be worth adding a section for "Version history"? Figma added this not that long ago https://help.figma.com/hc/en-us/articles/360038006754-View-a-file-s-version-history and there is support for tagging(naming) different versions. Also designers need to start using it and tagging versions, it can be helpful when you are working on the version 1.0.0 of the app, but the designer is already doing the design changes for the version 1.1.0 for example. If you think this is not being used yet much, I think this section can be skipped

Reply:
This looks really useful 👌 I added a proposal section about it and you can decide whether it should stay or not because I don’t have much insight into how often you use this in your projects at the moment. 😅

Reply:
>I don’t have much insight into how often you use this in your projects at the moment

Me neither but I think its not being used much, yet I think we should start using it on projects which have simultaneous active development & design.

What you wrote sounds good to me, I would keep it!